dissect/cstruct/types/structure.py

```diff
@@ -780,11 +783,10 @@ def _generate_structure__init__(fields: list[Field]) -> FunctionType:
     template: FunctionType = _make_structure__init__(len(field_names))
     return type(template)(
         template.__code__.replace(
-            co_consts=(None, *[field.type.__default__() for field in fields]),
-            co_names=(*field_names,),
+            co_names=(*chain.from_iterable(zip((f"__{name}_default__" for name in field_names), field_names)),),
```

Comment:
```suggestion
            co_names=tuple(chain.from_iterable(zip((f"__{name}_default__" for name in field_names), field_names)))
```

Maybe somewhat nicer without the dangling comma?
Or is it slower?

(it almost feels like lisp with all the parentheses, though I like the conciseness)